tests/sharding/client/sign/sign.go

```diff
@@ -171,7 +171,15 @@ func discoverRekorURLs(signingConfig *root.SigningConfig, url string) ([]string,
 	if url != "" {
 		return []string{url}, nil
 	}
-	return root.SelectServices(signingConfig.RekorLogURLs(), signingConfig.RekorLogURLsConfig(), []uint32{2}, now)
+	services, err := root.SelectServices(signingConfig.RekorLogURLs(), signingConfig.RekorLogURLsConfig(), []uint32{2}, now)
+	if err != nil {
+		return nil, err
+	}
+	var urls []string
+	for _, s := range services {
+		urls = append(urls, s.URL)
+	}
+	return urls, nil
```

Comment:
It would be less work to change the return type of this method to []root.Service and handle this in the caller but then the case above this (`url != ""`) becomes trickier :shrug: